tests/__mocks__/handlers.ts

```diff
@@ -38,12 +44,18 @@ export const handlers = [
   http.get("https://api.github.com/repos/ubiquibot/comment-incentives/issues/22/comments", () => {
     return HttpResponse.json(issue22CommentsGet);
   }),
+  http.get("https://api.github.com/repos/Meniole/conversation-rewards/issues/13/events", () => {
+    return HttpResponse.json(issue13EventsGet);
+  }),
   http.get("https://api.github.com/repos/ubiquity/work.ubq.fi/issues/69/comments", () => {
     return HttpResponse.json(issue69CommentsGet);
   }),
   http.get("https://api.github.com/repos/ubiquibot/comment-incentives/issues/25/comments", () => {
     return HttpResponse.json(issue25CommentsGet);
   }),
+  http.get("https://api.github.com/repos/Meniole/conversation-rewards/issues/13/comments", () => {
+    return HttpResponse.json(issue13CommentsGet);
+  }),
   http.get("https://api.github.com/repos/ubiquibot/comment-incentives/issues/22/timeline", () => {
     return HttpResponse.json(issueTimelineGet);
   }),
```

Comment:
Why not make these single line functions? 

```typescript
 http.get("https://api.github.com/repos/ubiquibot/comment-incentives/issues/22/timeline", () => HttpResponse.json(issueTimelineGet)
```

Reply:
Auto generated defaulted to this, can change if necessary.